src/main/java/lotto/domain/Lottos.java

```diff
@@ -27,4 +24,14 @@ public int quantity() {
         return lottos.get(index).getLottoNumbers();
     }
 
+    public void addLotto(Lotto lotto) {
+        if(lottos == null){
+            lottos = new ArrayList<>();
+        }
+        lottos.add(lotto);
+    }
+
+    public void addManualLottos(List<Lotto> manualLottos) {
```

Comment:
해당 메서드는 manualLottos 뿐만 아니라 모든 로또을 add 할 수 있을거 같아요! 😄 
```suggestion
    public void addLottos(List<Lotto> lottos) {
```